src/main/java/utility/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package utility;
+
+import java.util.Random;
+import model.Color;
+
+public class ColorSupplier {
```

Comment:
Avoid using static fields unless necessary. Here, the Random instance does not need to be static since each ColorSupplier object could have its own Random instance.